common/web/lm-worker/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@keymanapp/lm-worker",
+  "version": "15.0.233",
+  "description": "Keyman/Predictive Text worker",
+  "main": "build/index.js",
+  "scripts": {
+    "karma": "karma",
+    "mocha": "mocha",
+    "test": "gosh ./test/test.sh",
+    "build": "tsc -b",
+    "clean": "tsc -b --clean"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@keymanapp/models-types": "*",
+    "@keymanapp/resources-gosh": "*",
+    "@types/node": "^10.17.21",
+    "chai": "^4.3.4",
+    "karma": "^6.3.4",
+    "karma-browserstack-launcher": "^1.6.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-edge-launcher": "^0.4.2",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-fixture": "^0.2.6",
+    "karma-html2js-preprocessor": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-json-fixtures-preprocessor": "0.0.6",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-safari-launcher": "^1.0.0",
+    "karma-teamcity-reporter": "^1.1.0",
```

Comment:
Feels a little weird to have all of these in-browser testing dependencies here for the _worker-only_ project... but I guess we do have the unit tests that directly test its installation directly into WebWorkers, which would best be tested via an actual browser run.